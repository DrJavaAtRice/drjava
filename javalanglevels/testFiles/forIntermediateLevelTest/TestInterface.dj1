interface TestInterface1 {
  abstract int myMethod(); 
}

interface TestInterface2 extends TestInterface1 {
  
}