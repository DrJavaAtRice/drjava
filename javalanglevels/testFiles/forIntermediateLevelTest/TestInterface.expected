// Language Level Converter line number map: dj*->java. Entries: 7
//     1->3         2->4         3->5         4->6         5->7         6->8         7->9    
interface TestInterface1 {
  public abstract int myMethod(); 
}

interface TestInterface2 extends TestInterface1 {
  
}
